malebranche/client/profiler.py

```diff
@@ -0,0 +1,17 @@
+import cProfile
+import pstats
+
+
+class Profiler:
+    def __init__(self):
+        self.profiler = cProfile.Profile()
```

Comment:
Only a naming style detail: As the class is named `Profiler` (and you probably name instances as `profiler`) one option in this case is to rename the `profiler` attribute as `client` (or something similar) to avoid `profiler.profiler` accesings (maybe `profiler.client` or something similar will looks a bit fancier 🤙).

Reply:
Yep, you're right. I'm not sure whether `client` is a good choice, though :sweat:. I'll think about it and change it in a future issue.

Reply:
What do you think of `backend`? Since the `Profiler` class will become an interface soon working with different profiler implementations, this attribute represents the actual backend used to resolve operations. @garciparedes

Reply:
Yes, that's a good choice! Related with this topic, another libraries like `aiopg` use the word `impl` in these cases, but `backend` sounds better for me. 😉